.github/workflows/nuget.yml

```diff
@@ -0,0 +1,68 @@
+name: NuGet
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  ci-nuget:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include: [windows-latest, ubuntu-latest, macos-latest]
+    env:
+      DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+      DOTNET_CLI_TELEMETRY_OPTOUT: true
+      DOTNET_NOLOGO: true
+      NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }}
```

Comment:
This will require that @madler or the other contributor to create a nuget.org account and an api key and add it to their github secrets that github actions can then read.